src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,99 @@
+.clock {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 310px;
+  height: 310px;
+  border-radius: $border-radius;
+  border: 5px solid transparent;
+  animation: cose 3s ease-in;
+  box-shadow: inset 0 0 10px rgba(0, 0, 0, 0.25), 0 0 48px rgba(0, 0, 0, 0.35);
+  background: linear-gradient(#fffeef, #fffeef) padding-box,
+              linear-gradient(45deg, $mins-color, $secs-color, $clock-color) border-box;
+
+  &:hover {
+    transform: scale(1.1) rotate(1deg);
+    transform-origin: center center;
+
+  }
+
+  &__border {
+    position: relative;
+    width: 300px;
+    height: 300px;
+    border-radius: $border-radius;
+    border: 2px solid $clock-color;
+    animation: cose 3s ease-in;
+  }
+
+  &__middle {
+    position: absolute;
+    top: 45%;
+    left: 48%;
+    border: 2px solid $clock-color;
+    background-color: $clock-color;
+    border-radius: $border-radius;
+    width: 20px;
+    height: 20px;
+    z-index: 1;
+  }
+
+  &__time-3 {
+    position: absolute;
+    top: 43%;
+    left: 93%;
+  }
+
+  &__time-6 {
+    position: absolute;
+    top: 90%;
+    left: 48%;
+  }
+
+  &__time-9 {
+    position: absolute;
+    top: 43%;
+    left: 1%;
+  }
+
+  &__time-12 {
+    position: absolute;
+    left: 47%;
+  }
```

Comment:
in programming, it is called _Magic Numbers_ 😄
mostly, such values are not used, there should be some maths to calculate distances appropriately
just for you to know that it's not always the case to use such values in your code

Reply:
I'll fix those tomorrow. Thank you for helping.

Reply:
it is not obligatory in this case, just on your will